cypress/page-objects/pastPerfectPage/PastPerfectEle.js

```diff
@@ -0,0 +1,22 @@
+class PastPerfectEle {
```

Comment:
Could you please update the name of this class 
it is actually `PastPerfectConjugationPage`